packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6506,7 +6526,12 @@ export default class Meeting extends StatelessWebexPlugin {
     turnServerInfo?: TurnServerInfo
   ): Promise<void> {
     const LOG_HEADER = 'Meeting:index#addMedia():establishMediaConnection -->';
-    const isRetry = this.retriedWithTurnServer;
+    const isRetry = this.isMoveTo || this.retriedWithTurnServer;
```

Comment:
maybe `isReconnecting` would be a better name now for this variable? Because in the moveTo flow we are not retrying, we are just doing a new connection. Even `isReconnecting` is not a perfect name, but I can't think of a better one...